src/librustdoc/html/static/css/themes/light.css

```diff
@@ -68,7 +68,7 @@
 	--test-arrow-color: #f5f5f5;
 	--test-arrow-background-color: rgba(78, 139, 202, 0.2);
 	--test-arrow-hover-color: #f5f5f5;
-	--test-arrow-hover-background-color: #4e8bca;
+	--test-arrow-hover-background-color: rgb(78, 139, 202);
```

Comment:
I changed the color scheme in this file and in `dark.css` to make it closer to `--test-arrow-background-color` so it's more obvious they're linked.